tests/numerics/project_to_surface_03.output

```diff
@@ -9,7 +9,7 @@ DEAL::    Quad 4, projected point=0.000 0.000 0.000  (quad is from -0.250 0.000
 DEAL::    Quad 5, projected point=0.000 0.000 1.000  (quad is from 0.250 0.000 1.000 to 1.000 0.000 1.000 to -0.250 1.000 1.000 to 1.000 1.000 1.000)
 DEAL::
 DEAL::Trial point = 1.000 0.000 0.000
-DEAL::    Quad 0, projected point=-0.059 0.257 0.257  (quad is from -0.250 0.000 0.000 to 0.250 1.000 0.000 to 0.250 0.000 1.000 to -0.250 1.000 1.000)
+DEAL::    Quad 0, projected point=0.000 0.000 0.500  (quad is from -0.250 0.000 0.000 to 0.250 1.000 0.000 to 0.250 0.000 1.000 to -0.250 1.000 1.000)
```

Comment:
same here?

Reply:
Yes. This particular projection problem is not well-posed: there are multiple minimizers and the new version prefers the value on the edges.